iceoryx_hoofs/design/include/iox/static_lifetime_guard.hpp

```diff
@@ -23,8 +23,6 @@
 
 namespace iox
 {
-namespace design_pattern
-{
```

Comment:
Just to make sure, this class is public and should be added to the overview `Readme.md`?

Reply:
Correct, the class is public and I added it to the hoofs readme.